src/skottie-player.ts

```diff
@@ -3,6 +3,53 @@
  */
 
 import {LitElement, html, customElement, property, css} from 'lit-element';
+const LottieModule = (window as any).lottie;
+import * as lottie from 'lottie-web/build/player/lottie_light';  // Using light version for lighter bundle
+type LottiePlayerRoot = typeof lottie.default;
+const lottiePlayer = LottieModule as any as LottiePlayerRoot;
```

Comment:
lottiePlayer -> please check Typescript's naming conventions whether the first character of a constant's name should be capitalized. This const seems to have conflicting capitalization with the other const at Line 6.